### PR TITLE
make navbar responsive

### LINKED ISSUE #60: Unresponsive navbar
### What bug did you encounter?

The navigation section of  site is appealing as things appear cramped up

### What did you expect to happen?

I expected to find a navigation menu 

### Screenshots

Screenshot_2022-10-03-07-33-14-859_com android chrome

### Additional information

I'll love to take this on if assigned to me.

### PR DESCRIPTION
**Pull request by Yushi5058**
# Pull Request

## Description:

Resolve unresponsive navbar in mobile.

## Related Issue:

Closes #60

### CODE REVIEW

File: src/app/nav/nav.component.css [hunk 1]
```diff
@@ -21,4 +21,19 @@ nav {
...
+font-size: .9rem;
+min-width:20%;
+max-width:40%;
+padding-right:100px;
+}
+a[routerLink="/mission"] {
+  padding-left:20px;
+}
+
+}
```
Comment (naomi-lgbt) [inline src/app/nav/nav.component.css:39]:
```suggestion
@media (min-width: 360px) {
  .navlinks {
    justify-content: center;
    gap: 15px;
    font-size: 0.9rem;
    min-width: 20%;
    max-width: 40%;
    padding-right: 100px;
  }
  
  a[routerLink="/mission"] {
    padding-left: 20px;
  }
}
```

Let's format this.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]
banger pr, thanks for contributing!

**Comment by naomi-lgbt** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

You can also see all open issues [through our contributor tool](https://contribute.naomi.lgbt)

**Comment by Yushi5058** [comment]
> banger pr, thanks for contributing!

Your welcome ^^

**Pull request merged**